modules/features/player/src/main/java/au/com/shiftyjelly/pocketcasts/player/viewmodel/BookmarksViewModel.kt

```diff
@@ -266,18 +265,12 @@ class BookmarksViewModel
         }
     }
 
-    fun onShareClicked(fragmentManager: FragmentManager) {
-        (_uiState.value as? UiState.Loaded)?.let {
-            val bookmark = it.bookmarks.firstOrNull { bookmark -> multiSelectHelper.isSelected(bookmark) }
-            bookmark?.let {
-                viewModelScope.launch {
-                    val podcast = podcastManager.findPodcastByUuidSuspend(bookmark.podcastUuid) ?: return@launch
-                    val episode = episodeManager.findEpisodeByUuid(bookmark.episodeUuid) as? PodcastEpisode ?: return@launch
-                    ShareDialogFragment
-                        .newInstance(podcast, episode, sourceView)
-                        .show(fragmentManager, "share_dialog")
```

Comment:
I made a mistake when refactoring in #2499. This should open bookmark directly instead of showing sharing dialog. I changed it back in this PR.